### ISSUE TITLE
Files screen: add pinning option to bottom bar when multiple files selected

### ISSUE THREAD

**Issue by jessicaschilling** [ipfs-shipyard/ipfs-webui#1682]
The third and fourth panes in the [pinning service integration Files screen mockups](https://www.figma.com/file/4cYoTNbjnnV7I7qeOEhjg8/WebUI-Desktop-Pinning-Hi-Fi-Screens%3A-Files?node-id=0%3A1) depict a workflow for setting pinning on multiple files. However, this was de-scoped from pinning integration MVP due to unexpected complexity (orig https://github.com/ipfs-shipyard/ipfs-webui/issues/1500#issuecomment-717483195):

- when user selects multiple items a bottom drawer is displayed with operations that could be applied to all of them
  - Q: should we support bulk pin control when multiple files are selected this way?
   - A: we could, but **not in scope for MVP**. 
     - Rationale: this is additional work which needs more thought. We could either:
       -  pin each selected item separately, 
       - or we could follow logic used by _Share_ action and create ephemeral directory that contains selected items and only pin the root CID

This issue exists to isolate and implement that work at a later date.
cc @rafaelramalho19 @lidel 

Related request from @tcll to implement, regardless of pinning services: https://github.com/ipfs-shipyard/ipfs-webui/issues/1681

**Comment by lidel** [comment]
Quick feedback after playing with stage 1 (https://github.com/ipfs-shipyard/ipfs-webui/pull/1685) implementation:  
- I believe we should pin each selected item separately

Rationale: the way we will do remote pin status check is to:
1. check if file is pinned
2. check if any of its parent directories is pinned 

Due to this we can't do what we do for Share (create ephemeral directory with selected files and only pin the directory's CID). That would make it impossible to detect remote pin.


Just for reference, below is current UI from #1685. 
For MVP we should enable _Set pinning_  option and make it issue `ipfs.pin.remote.add` call for each CID separately.

![2020-11-06--21-32-35](https://user-images.githubusercontent.com/157609/98683631-8ed62500-2365-11eb-8d13-c5de25068a62.png)

**Comment by jessicaschilling** [comment]
Makes sense, and probably hews closest to most user's mental model of how this would actually work anyway - the ephemeral directory thing is a bit hard to wrap one's head around if you're thinking within the context of other ordinary file systems.

**Comment by SgtPooki** [comment]
need to confirm whether this was already addressed by https://github.com/ipfs/ipfs-webui/pull/1919
